package.json

```diff
@@ -83,6 +83,7 @@
     "@types/puppeteer": "^1.11.1",
     "@types/resize-observer-browser": "^0.1.2",
     "@types/three": "^0.93.12",
+    "@types/webgl2": "^0.0.5",
```

Comment:
That's interesting. What are these being used for?

Reply:
I remember you found this fix for me when I upgraded three and got type errors. Three added it to their dependencies; maybe I can just inherit it directly now?

Reply:
Oh yah, forgot about that. If it isn't in r108, then we can't benefit from their fix. Of course, we could also update to r110...

Reply:
I did update to r110 here, as I needed my own bugfix. I'll see if I can remove this now.

Reply:
I get this error when building if I remove it: node_modules/three/src/renderers/webgl/WebGLUtils.d.ts:3:43 - error TS2304: Cannot find name 'WebGL2RenderingContext'.

3  constructor( gl: WebGLRenderingContext | WebGL2RenderingContext, extensions: any, capabilities: any );